support-internationalisation/version.sbt

```diff
@@ -1 +1 @@
-version := "0.16-SNAPSHOT"
```

Comment:
I ran a test release of support-internationalisation to make sure my local setup was working, this is the result of that